packages/composer.rb

```diff
@@ -25,7 +25,7 @@ def self.preinstall
 
   def self.install
     downloader "https://github.com/composer/composer/releases/download/#{version}/composer.phar",
```

Comment:
Use `source_url`?

Reply:
Does that work with `.phar` files?

Reply:
Yes, any non-archive files should work

Reply:
Done.  I also abstracted the path from `~/.bashrc`.